app/assets/stylesheets/application.scss

```diff
@@ -573,3 +573,7 @@ table {
 .masked-input {
   font-family: 'AllBullets';
 }
+
+.code-in-table {
+  font-family: DejaVu Sans Mono, Courier New, monospace;
```

Comment:
Because I was hoping to retire without decent CSS knowledge. It does not work...